src/assets/javascript/main.js

```diff
@@ -0,0 +1,60 @@
+import "../../assets/main.css"; 
+const Button = require("../../components/button");
+document.querySelector("#Buttons").insertAdjacentHTML("beforeend", Button());
+
+const inputReuslt = document.querySelector(".js-result");
+const patternElements = document.querySelectorAll(".js-btn-pattern");
+const ClearAllElement = document.querySelector(".js-btn-ClearAll");
+const lessMoreElement = document.querySelector(".js-btn-less-more");
+const delElement = document.querySelector(".js-btn-Del");
+const equaElement = document.querySelector(".js-btn-equal");
+
+//-------------------- textContent of all buttons-----------------------------//
+ClearAllElement.textContent = "C";
+lessMoreElement.textContent = "+/-";
+delElement.textContent = "«";
+equaElement.textContent = "=";
+const actions_patterButtons = [
+	"%","7","8","9",
+	"/","4","5","6",
+	"*","1","2","3",
+	"+","-",".","0",];
+
+for (let j = 0; j < patternElements.length; j++) {
```

Comment:
i just saw it now, that i don't need this  `for...loop`, i can just put this line :`patternElements[j].textContent = actions_patterButtons[j];` inside the main `for...lop`

Reply:
i already changed this